Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM frolvlad/alpine-python3
```

Comment:
also, I forgot that we currently need to add the following for ardere to work:  $ apt-get install -y wget;